lib/Canvas.js

```diff
@@ -46,7 +62,7 @@ module.exports = {
     const stream = csv.format({ headers: true })
     stream.pipe(writer)
 
-    enrollments.forEach(e => stream.write(e))
```

Comment:
Look! We check if the enrollments are valid (via `validEnrollments`) but then we use `enrollments` to write to the CSV file! 🤦‍♂